src/config/index.js

```diff
@@ -31,7 +31,9 @@ export const functions = firebase.functions();
 let firebase_messaging;
 if (firebase.messaging.isSupported()) {
   firebase_messaging = firebase.messaging();
-  firebase_messaging.usePublicVapidKey(process.env.REACT_APP_FIREBASE_FCM_VAPID_KEY);
```

Comment:
You really need to consider this.

Reply:
@shehand sorry for this problem, but couldn't fixed this issue in this PR , But after #141  this problem solved . I tried the fix mentioned in the stackoverflow but it didn't worked on those pr which are already created .  That fix is working on new branch I am creating. This firebase config will not bother anymore in future PRs.